src/Header.php

```diff
@@ -16,226 +16,1226 @@ final class Header
+     * @var string
+     * @see https://tools.ietf.org/html/rfc7234#section-5.5
+     */
+    const WARNING = 'Warning';
+
+    /**
+     * @var string
+     * @see https://tools.ietf.org/html/rfc7034
+     */
+    const X_FRAME_OPTIONS = 'X-Frame-Options';
 
     // Provisional Message Headers
 
-    /** @deprecated */
-    const ACCESS_CONTROL = 'Access-Control'; // W3C Web Application Formats WG
-    const ACCESS_CONTROL_ALLOW_CREDENTIALS = 'Access-Control-Allow-Credentials'; // W3C Web Application Formats WG
-    const ACCESS_CONTROL_ALLOW_HEADERS = 'Access-Control-Allow-Headers'; // W3C Web Application Formats WG
-    const ACCESS_CONTROL_ALLOW_METHODS = 'Access-Control-Allow-Methods'; // W3C Web Application Formats WG
-    const ACCESS_CONTROL_ALLOW_ORIGIN = 'Access-Control-Allow-Origin'; // W3C Web Application Formats WG
-    const ACCESS_CONTROL_MAX_AGE = 'Access-Control-Max-Age'; // W3C Web Application Formats WG
-    const ACCESS_CONTROL_REQUEST_METHOD = 'Access-Control-Request-Method'; // W3C Web Application Formats WG
-    const ACCESS_CONTROL_REQUEST_HEADERS = 'Access-Control-Request-Headers'; // W3C Web Application Formats WG
-    const COMPLIANCE = 'Compliance'; // RFC4229
-    const CONTENT_TRANSFER_ENCODING = 'Content-Transfer-Encoding'; // RFC4229
-    const COST = 'Cost'; // RFC4229
-    const EDIINT_FEATURES = 'EDIINT-Features'; // RFC6017
-    const MESSAGE_ID = 'Message-ID'; // RFC4229
-    /** @deprecated */
-    const METHOD_CHECK = 'Method-Check'; // W3C Web Application Formats WG
-    /** @deprecated */
-    const METHOD_CHECK_EXPIRES = 'Method-Check-Expires'; // W3C Web Application Formats WG
-    const NON_COMPLIANCE = 'Non-Compliance'; // RFC4229
-    const OPTIONAL = 'Optional'; // RFC4229
-    /** @deprecated */
-    const REFERER_ROOT = 'Referer-Root'; // W3C Web Application Formats WG
-    const RESOLUTION_HINT = 'Resolution-Hint'; // RFC4229
-    const RESOLVER_LOCATION = 'Resolver-Location'; // RFC4229
-    const SUBOK = 'SubOK'; // RFC4229
-    const SUBST = 'Subst'; // RFC4229
-    const TITLE = 'Title'; // RFC4229
-    const UA_COLOR = 'UA-Color'; // RFC4229
-    const UA_MEDIA = 'UA-Media'; // RFC4229
-    const UA_PIXELS = 'UA-Pixels'; // RFC4229
-    const UA_RESOLUTION = 'UA-Resolution'; // RFC4229
-    const UA_WINDOWPIXELS = 'UA-Windowpixels'; // RFC4229
-    const VERSION = 'Version'; // RFC4229
-    const X_DEVICE_ACCEPT = 'X-Device-Accept'; // W3C Mobile Web Best Practices WG
-    const X_DEVICE_ACCEPT_CHARSET = 'X-Device-Accept-Charset'; // W3C Mobile Web Best Practices WG
-    const X_DEVICE_ACCEPT_ENCODING = 'X-Device-Accept-Encoding'; // W3C Mobile Web Best Practices WG
-    const X_DEVICE_ACCEPT_LANGUAGE = 'X-Device-Accept-Language'; // W3C Mobile Web Best Practices WG
-    const X_DEVICE_USER_AGENT = 'X-Device-User-Agent'; // W3C Mobile Web Best Practices WG
+    /**
+     * @var string
+     * @deprecated
+     */
+    const ACCESS_CONTROL = 'Access-Control';
+
+    /**
+     * @var string
```

Comment:
There is already  specification for the access-control headers, which could be linked: https://fetch.spec.whatwg.org/#http-requests or https://fetch.spec.whatwg.org/#http-responses

Reply:
added